stackrox/base/serviceaccounts/kustomization.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Scope these SAs to a namespace.
```suggestion
kind: Kustomization
namespace: stackrox
```